query/control/controller_test.go

```diff
@@ -145,21 +143,6 @@ func TestController_QuerySuccess(t *testing.T) {
 			if err := q.Err(); err != nil {
 				t.Errorf("unexpected error: %s", err)
 			}
-
-			stats := q.Statistics()
-			if stats.CompileDuration == 0 {
-				t.Error("expected compile duration to be above zero")
-			}
-			if stats.QueueDuration == 0 {
-				t.Error("expected queue duration to be above zero")
-			}
-			if stats.ExecuteDuration == 0 {
-				t.Error("expected execute duration to be above zero")
-			}
-			if stats.TotalDuration == 0 {
-				t.Error("expected total duration to be above zero")
-			}
-			validateRequestTotals(t, reg, 1, 0, 0, 0)
```

Comment:
Don't we still want to call `validateRequestTotals` at the end?

Reply:
Yes - on second look we definitely do! thanks!